lockup/README.md

```diff
@@ -1,43 +1,64 @@
 # Lockup / Vesting contract
 
-# PLEASE WAIT UNTIL [ISSUE 141](https://github.com/near/core-contracts/issues/141) IS RESOLVED BEFORE DEPLOYING THE CONTRACT
-
 ## Overview
 
-This contract acts as an escrow that locks and holds an owner's tokens for a lockup period. A lockup period starts
-from the moment transfers are enabled by voting and lasts for the specified duration.
+This contract acts as an escrow that locks and holds an owner's tokens for a lockup period. 
+The contract contains of lockup and vesting processes that go simultaneously.
+
+A lockup period starts from the specified timestamp (or, if not specified, from moment when transfers are enabled by voting) 
+and lasts for the specified duration.
 It's also possible to lock tokens until the absolute timestamp. In this case the tokens will be available after the
```

Comment:
> It's also possible to lock tokens until the absolute timestamp.

I don't believe that is true.

Reply:
I think it's true, you could set `lockup_timestamp` to desired absolute timestamp and set `release_duration` to 0.
But it looks like this sentence adds nothing to the meaning, it could be deleted not to confuse anyone